src/lib/errors.ts

```diff
@@ -0,0 +1,51 @@
+import { Types } from '../snarky/types.js';
+import { TokenId } from './account_update.js';
+import { Int64 } from './int.js';
+
+export { invalidTransactionError };
+
+function invalidTransactionError(
+  transaction: Types.ZkappCommand,
+  errors: string[][][],
+  { accountCreationFee }: { accountCreationFee: string | number }
+) {
+  let lastUpdate = errors[errors.length - 1];
+
+  if (lastUpdate.some(([msg]) => msg === 'Invalid_fee_excess')) {
```

Comment:
Minor nit but can we refactor `invalidTransactionError` a bit so that `Invalid_fee_excess` is less coupled from the function, so it's easier to add more errors later down the road?